google/cloud/monitoring_v3/gapic/alert_policy_service_client.py

```diff
@@ -245,7 +302,8 @@ def list_alert_policies(
             >>>
             >>> client = monitoring_v3.AlertPolicyServiceClient()
             >>>
-            >>> name = client.project_path('[PROJECT]')
```

Comment:
These resource helper methods being removed is a breaking change, and this library is GA. If another release needs to be made, glue these back in with synthtool.